src/test/java/org/folio/event/handler/InventoryCreateAsyncRecordHandlerTest.java

```diff
@@ -95,12 +112,51 @@ var record = createKafkaRecord(eventObject, DIKU_TENANT);
     });
   }
 
+  @Test
+  void positive_shouldSkipInventoryCreateEventIfCentralOrderingIsDisabled() {
+    var eventObject = createResourceEvent(DIKU_TENANT, CREATE);
+    var record = createKafkaRecord(eventObject, DIKU_TENANT);
+    doReturn(Future.succeededFuture(Response.ok(createSettingCollection()).build()))
```

Comment:
we can move createSettingCollection() to variable 
var emptySetting = new SettingCollection(); it is more easier to understand

Reply:
Replaced, thanks